demos/tutorials/6_fluid_tank.ipynb

```diff
@@ -0,0 +1,280 @@
+        "\n",
+        "The fluid container is a cylinder of radius $R_\\mathrm{tank}$ and height $h_\\mathrm{tank}$. The container is initially partially filled. New fluid is injected with velocity $v_\\mathrm{inlet}$ into the tank from a circular inlet at the top of the container with arbitrary coordinates $(x_\\mathrm{inlet}, y_\\mathrm{inlet})$. The fluid exits the tank from a cylindrical outlet at the bottom of the container.\n",
```

Comment:
"Our exploration will focus in the particular case of a cylindrical fluid tank that is defined by a base radius $R_\\mathrm{tank}$ and height $h_\\mathrm{tank}."

We should fix the word tank and use it throughout instead of container.

Reply:
Should it be *cylindrical inlet?

Reply:
> Should it be *cylindrical inlet?

Nop. The inlet has no height. It's a circle.

Reply:
Alright!

Ok, but the outlet is cylindrical?

Reply:
Yep.